layouts/partials/data_jobs/setup-platforms.html

```diff
@@ -1,50 +1,68 @@
 {{ $dot := . }}
 <div class="dsm-containers">
   <div class="container cards-dd">
-    <div class="row row-cols-1 row-cols-md-5 g-2 g-xl-3 justify-content-sm-center">
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/emr">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/amazon_emr.png" "class" "img-fluid" "alt"
-            "Amazon EMR" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/databricks/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/databricks.png" "class" "img-fluid" "alt" "Databricks" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/dataproc/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/google_cloud_dataproc.png" "class"
-            "img-fluid" "alt" "GCP Dataproc" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/kubernetes/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/kubernetes.png" "class" "img-fluid" "alt"
-            "Kubernetes" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/airflow/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/airflow.png" "class" "img-fluid" "alt" "Airflow" "width" "200") }}
-          </div>
-        </a>
-      </div>
+    <div class="row row-cols-1 row-cols-md-4 g-2 g-xl-3 justify-content-sm-center">
+
+     <div class="col">
```

Comment:
@rtrieu  can we reorganize this setup section now that we have more technologies that aren't Spark and the logos don't make that clear?

Specifically i'm thinking
```
Data Jobs Monitoring supports multiple job technologies. To get started, select your technology and follow the installation instructions:

- Logo list of Databricks, Airflow, dbt

Apache Spark jobs on the following platforms:
- Logo list of K8s, EMR, Google Dataproc

```

<img width="772" height="484" alt="Screenshot 2025-12-24 at 1 33 43 PM" src="https://github.com/user-attachments/assets/b16f60a6-5d7d-4166-a292-1051bfff70dd" />